Bio/Entrez/Parser.py

```diff
@@ -451,7 +451,7 @@ def parse_xsd(self, root):
                 self.lists.append(name)
 
     def elementDecl(self, name, model):
-        """Callback function is called for each element declaration in a DTD.
+        """Return element according to its declaration in a DTD.
```

Comment:
I don't think "Return ..." is correct for a call-back function. Maybe: "This is a call-back function for element declaration in a DTD"?

Reply:
Your suggestion is not imperative, therefore `pydocstyle` does not like it. 
I would opt for "Call a call-back function for each element declaration in a DTD", but I'll wait for the module owners opinion.

Reply:
I'm not 100% sure if this method itself is the callback function or not, and that is important for the phrasing (and meaning).